torch/csrc/jit/codegen/cuda/kernel_db/test/test_nvfuser_kernel_db_open.cpp

```diff
@@ -0,0 +1,123 @@
+#if defined(USE_CUDA)
+#include <gmock/gmock-matchers.h>
+#include <gtest/gtest.h>
+
+#include <torch/torch.h>
+
+#include <torch/csrc/jit/codegen/cuda/kernel_db/kernel_db.h>
+#include <torch/csrc/jit/codegen/cuda/kernel_db/utils.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_gpu_validator.h>
+#include <torch/csrc/jit/codegen/cuda/test/test_utils.h>
+
+// Tests go in torch::jit
+namespace torch {
+namespace jit {
+
+namespace Nvf = torch::jit::fuser::cuda;
+
+// RUN CMD: bin/test_jit --gtest_filter="NVFuserTest*KernelDb_Open*"
+TEST_F(NVFuserTest, KernelDb_Open_CUDA) {
+  // Try a DB path that should fail
+  try {
+    auto& kernel_db = Nvf::KernelDb::get("../nvfuser_kernel_db_test", "db.csv");
```

Comment:
We have both kernel_db_open & kernel_db_write tests using the same `db.csv` file for testing. If we ended up running tests in parallel, there's a race condition on this one. Let's change `db.csv` to '__FUNCTION__` maybe?

Reply:
I am not sure what you mean be `FUNCTION`.  I can just give them separate directories so there isn't any conflict.

Reply:
sorry it's the formatting thing. I mean a unique name for each and was suggesting the macro `__FUNCTION__`.

Separate directories would also work~

Reply:
I added separate directories for now.